jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -192,8 +192,8 @@ public StepsRunner runPushImageStep() {
   }
 
   public StepsRunner runBuildTarballAndLoadDockerStep() {
```

Comment:
Should this method be renamed?  Hmm, I guess not as it will need to build the tarball when that is separated.

Reply:
Yeah good catch, this should probably be renamed. We ended up agreeing on just creating a separate step instead of splitting this one up - https://github.com/GoogleContainerTools/jib/pull/502#discussion_r200652779